### PR TITLE
Update badges in readme

### PR DESCRIPTION
**Pull request by MattIPv4**
A small PR just to update the badges in the readme file.

remove tip4commit - has sat at 0 for a very long time
remove getbadges - invalid ssl, badge no longer works, site seems rather abandoned
add commit activity - an interesting stat badge to have on the readme, showing how active cdnjs is
move license badge - move to before build related badges, with other "repo" badges and colour co-ordinated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-[![license](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat)](https://github.com/cdnjs/cdnjs/blob/master/MIT-LICENSE)
-[![tip for next commit](https://tip4commit.com/projects/919.svg)](https://tip4commit.com/github/cdnjs/cdnjs) [![Bountysource](https://www.bountysource.com/badge/team?team_id=11914&style=bounties_posted)](https://www.bountysource.com/teams/cdnjs/bounties?utm_source=cdnjs&utm_medium=shield&utm_campaign=bounties_posted)
-[![GetBadges Game](https://cdnjs-cdnjs.getbadges.io/shield/company/cdnjs-cdnjs)](https://cdnjs-cdnjs.getbadges.io/?ref=shield-game)
+[![Dependency Status](https://img.shields.io/david/cdnjs/cdnjs.svg)](https://david-dm.org/cdnjs/cdnjs)
+[![devDependency Status](https://img.shields.io/david/dev/cdnjs/cdnjs.svg)](https://david-dm.org/cdnjs/cdnjs#info=devDependencies)
+[![Bountysource](https://www.bountysource.com/badge/team?team_id=11914&style=bounties_posted)](https://www.bountysource.com/teams/cdnjs/bounties?utm_source=cdnjs&utm_medium=shield&utm_campaign=bounties_posted)
+[![Commit Activity](https://img.shields.io/github/commit-activity/y/cdnjs/cdnjs.svg?style=flat)](https://github.com/cdnjs/cdnjs/commits/)
```
[the remark this answers is not in the document]
Comment (MattIPv4) [reply README.md]: Is there any reason for it to be separated, the PR is just a general cleanup of the readme badges?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/cdnjs/cdnjs/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3AHacktoberfest%20-label%3A%22in%20progress%22%20)
+[![License](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://github.com/cdnjs/cdnjs/blob/master/LICENSE)
 [![Drone CI](https://ci.cdnjs.com/api/badges/cdnjs/cdnjs/status.svg?branch=master)](https://ci.cdnjs.com/cdnjs/cdnjs)
-[![Dependency Status](https://david-dm.org/cdnjs/cdnjs.svg?theme=shields.io)](https://david-dm.org/cdnjs/cdnjs) [![devDependency Status](https://david-dm.org/cdnjs/cdnjs/dev-status.svg?theme=shields.io)](https://david-dm.org/cdnjs/cdnjs#info=devDependencies)
-[![license](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat)](https://github.com/cdnjs/cdnjs/blob/master/MIT-LICENSE)
-[![tip for next commit](https://tip4commit.com/projects/919.svg)](https://tip4commit.com/github/cdnjs/cdnjs) [![Bountysource](https://www.bountysource.com/badge/team?team_id=11914&style=bounties_posted)](https://www.bountysource.com/teams/cdnjs/bounties?utm_source=cdnjs&utm_medium=shield&utm_campaign=bounties_posted)
```
[the remark this answers is not in the document]
Comment (MattIPv4) [reply README.md]: Would a custom one without the current tip (made via shields.io) be viable instead?
example new

Currently the tip4commit badge is smaller and has the plastic style design which does not match all the other badges, sticking out like an unattractive sore thumb.
current

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PeterDaveHello) [reply README.md]: I prefer to separate different changes in different commits, which help we trace the changes in the future.

[the remark this answers is not in the document]
Comment (MattIPv4) [reply README.md]: As we're adding back the tip4commit, this PR is literally just to re-order and update all the badges to shields.io?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (PeterDaveHello) [reply README.md]: Sure!

**Comment by MattIPv4** [comment]
Please see #13107 (new PR with new branch with changes split up in different commits).

**Pull request closed without merging**